scripts/get-next-version.js

```diff
@@ -102,9 +103,11 @@ if (require.main !== module) {
 (async () => {
   process.chdir(path.join(__dirname, '..'))
 
-  const { nextVersion } = await getNextVersionForBinary()
+  const args = minimist(process.argv.slice(2))
 
-  if (process.argv.includes('--npm') && checkedInBinaryVersion !== nextVersion) {
+  const nextVersion = args.nextVersion || (await getNextVersionForBinary()).nextVersion
```

Comment:
can we add a test for passing in the `nextVersion` as an arg explicitly to https://github.com/cypress-io/cypress/blob/develop/scripts/unit/get-next-version-spec.js?